src/core_modules/capture-core/components/Pages/NewEvent/DataEntry/Assignee/Assignee.component.js

```diff
@@ -21,31 +27,25 @@ const getStyles = () => ({
 
 type Props = {
     classes: Object,
+    orientation: string
 };
 
 const Assignee = (props: Props) => {
-    const { classes, ...passOnProps } = props;
-
+    const { classes, orientation, ...passOnProps } = props;
     return (
-        <div
-            className={classes.container}
+        <div className={classes[classNames({
+            container: orientation === 'horizontal',
+            containerVertical: orientation === 'vertical',
+        })]}
         >
```

Comment:
This felt unnecessary complex and a bit weird since we can't concat different classes this way. I would just use the ternary operator (if you feel the urge to use classNames, use it in the regular way). I'm adding this comment only, but this applies to all usage of classNames in this file.

Reply:
Yeah was complex for me as well for sure. Will follow what you suggested. :)